recipe_scrapers/weightwatchers.py

```diff
@@ -0,0 +1,152 @@
+
+    def difficulty(self):
+        return self._extractItemField(self._findDataContainer().contents[4]).get_text()
+
+    #   Alternative way to extract data based on description instead of position
+    #    def total_time(self):
+    #        return get_minutes(
+    #            self.__findDataContainer()
+    #            .find("div", string=re.compile(r"minutes Total Time"))
+    #            .previous_sibling
+    #        )
+
+    def image(self):
+        backgroundImgStyle = self.soup.find("div", {"class": "styles_image__2dnNm"})[
+            "style"
+        ]
+
+        if backgroundImgStyle:
+            return (
+                re.compile(r'url\("(?P<imgurl>\S*)"\);')
+                .search(backgroundImgStyle)
+                .groupdict()
+                .get("imgurl")
+            )
+
+        return None
+
+    def _findIngridientTags(self):
+        return self.soup.find(
+            "h3", {"id": "food-detail-recipe-ingredients-header"}
+        ).parent.find_all("div", {"class": "styles_name__1OYVU"})
+
+    def _extractIngridientName(self, ingridient):
+        return normalize_string(
+            ingridient.find("div", {"class": "styles_ingredientName__1Vffd"})
+            .find("div")
+            .get_text()
+        )
+
+    def _extractPortionParts(self, ingridient):
+        tags = ingridient.find("div", {"class": "styles_portion__2NQyq"}).find_all(
+            "span"
+        )
+
+        return (
+            normalize_string(tags[0].get_text()),
+            normalize_string(tags[1].get_text()),
+            normalize_string(tags[2].get_text().replace(", ", "")) if tags[2] else None,
+        )
+
+    def __parseIngridient(self, ingridient):
+        ingridientName = self._extractIngridientName(ingridient)
+        amount, unit, comment = self._extractPortionParts(ingridient)
+
+        if comment:
+            return amount + " " + unit + " " + ingridientName + "; " + comment
+        else:
+            return amount + " " + unit + " " + ingridientName
+
+    def ingredients(self):
+        result = []
+        ingridientTags = self._findIngridientTags()
+
+        for ingridient in ingridientTags:
+            result.append(self.__parseIngridient(ingridient))
+        return result
+
+    def _getInstructions(self, headertag, headerattribute, headervalue, instructiontag):
+        instructions = self.soup.find(
+            headertag, {headerattribute: headervalue}
+        ).parent.find("ol")
+        return "\n".join(
+            [
+                normalize_string(instruction.get_text())
+                for instruction in instructions.find_all(instructiontag)
+            ]
+        )
+
+    def instructions(self):
+        return self._getInstructions(
+            "h3", "id", "food-detail-recipe-instruction-header", "div"
+        )
+
+    def description(self):
+        return self.soup.find("div", {"class": "copy-1"}).get_text().strip()
+
+    def nutrients(self):
```

Comment:
Nutrients should return a `Dict[str, str]` similar to the [this](https://github.com/hhursev/recipe-scrapers/blob/3ca85684c98546404d643c89d4dd897042cf523a/tests/test_schemaorg.py#L28).